src/main/java/io/anserini/rerank/lib/RocchioReranker.java

```diff
@@ -52,19 +52,25 @@ public class RocchioReranker implements Reranker {
   private final Analyzer analyzer;
   private final String field;
 
-  private final int fbTerms;
-  private final int fbDocs;
+  private final int topfbTerms;
+  private final int topfbDocs;
+  private final int bottomfbTerms;
+  private final int bottomfbDocs;
   private final float alpha;
   private final float beta;
+  private final float gamma;
   private final boolean outputQuery;
 
-  public RocchioReranker(Analyzer analyzer, String field, int fbTerms, int fbDocs, float alpha, float beta, boolean outputQuery) {
+  public RocchioReranker(Analyzer analyzer, String field, int topfbTerms, int topfbDocs,int bottomfbTerms, int bottomfbDocs, float alpha, float beta, float gamma, boolean outputQuery) {
```

Comment:
missing space before `bottomfbTerms`.

Actually, based on Java convention, should be `topFbTerms` and `bottomFbTerms` - proper camel case.

Reply:
change throughout please.